src/store/attributeModules/waterQualityModule.ts

```diff
@@ -24,29 +31,20 @@ export interface IDepthSettings {
 }
 
 @Module({ generateMutationSetters: true })
-export class WaterQualityModule extends VuexModule implements IAttributeModuleWithOptions {
-
+export class WaterQualityModule
+  extends VuexModule
+  implements IAttributeModuleWithOptions
+{
   public name = '$waterQuality';
   public isSelected = false;
   public loading = false;
-  public language = 'en';
+  public language: string | null = null;
   public selectedIds: number[] = [];
   public data: IResponseFormat[] | null = null;
   public siteTypes = [SiteTypes.Vesla];
   public selectedDepth: IDepthSettings = { option: DepthOptions.AllLayers };
   public options: IWaterQualityOption[] = [];
-
-  get availableOptions() {
-    let options: IAttributeOption[] = [];
-    if (this.language === 'fi') {
-      options = this.options.map((o) => ({ id: o.id, name: o.name_fi, online: true }));
-    } else if (this.language === 'sv') {
-      options = this.options.map((o) => ({ id: o.id, name: o.name_sv, online: true }));
-    } else {
-      options = this.options.map((o) => ({ id: o.id, name: o.name_en, online: true }));
-    }
-    return options.sort((a, b) => alphabeticCompare(a.name, b.name));
-  }
```

Comment:
Tää availableOptions ei enää toiminut kun Vue 3:ssa on tällanen juttu että toi api-kutsu palauttikin Proxy-objektin, kts. esim. https://stackoverflow.com/questions/64917686/vue-array-converted-to-proxy-object